### ISSUE TITLE
'@override' problem?

### ISSUE THREAD

**Issue by renmm** [jsdoc3/jsdoc#1215]
code:

``` javascript
/**
 * @classdesc Abstract class representing a network connection.
 * @alias Connection
 * @class
 */
function Connection() {}

/**
 * Open the connection.
 * 
 * @param {string} connection connection string
 */
Connection.prototype.open = function(connection) {
    // ...
};


/**
 * @classdesc Class representing a socket connection.
 * @alias Socket
 * @class
 * @augments Connection
 */
function Socket() {}

/**
 * Open the socket.
 *
 * @param {Object} options configuration option
 * @param {string} options.connection connection string
 * @param {string} options.userId user Id
 * @param {string} options.password user password
 * @override
 */
Socket.prototype.open = function(options) {
    // ...
};
```

generate document:

sorry, effect chart upload failed.

problem：

When the class to override the parent class method, the parameter name is not the same, why in the document can not see the parameters of sub class method ?

**Comment by renmm** [comment]
This is the document：

![image](https://cloud.githubusercontent.com/assets/2681624/15562863/cb71dab6-2335-11e6-828a-d731a1e9e511.png)

**Comment by AaronShi32** [comment]
I met the same problem when the @since tag use on subclasss method which have @override tag.
The result is @since value follow parent override method

**Comment by englercj** [comment]
This is also a problem for me when generating Typescript from jsdoc comments. The overridden method has incorrect params.

**Comment by hegemonic** [comment]
Fixed on master. The fix will be included in JSDoc 3.6.0.

For anyone who has a project that uses `@override` tags: I'd really appreciate it if you could test the fix and file new issues if you run into problems.
